UI/window-basic-filters.cpp

```diff
@@ -445,6 +452,28 @@ void OBSBasicFilters::UpdateFilters()
 	main->SaveProject();
 }
 
+void OBSBasicFilters::UpdateSplitter()
+{
+	bool show_splitter_frame =
+		ui->asyncFilters->count() + ui->effectFilters->count() > 0;
+	UpdateSplitter(show_splitter_frame);
+}
+
+void OBSBasicFilters::UpdateSplitter(bool show_splitter_frame)
+{
+	bool show_splitter_handle = show_splitter_frame;
+	uint32_t caps = obs_source_get_output_flags(source);
+	if ((caps & OBS_SOURCE_VIDEO) == 0)
```

Comment:
This is the same check as the constructor to check if this source is audioOnly.

Not sure if it makes more sense to store that info when it's checked in the constructor or have this check happen again here